lib/routes/movie_detail.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'about_us.dart';
+import '/widgets/error_message.dart';
+import '/model/genre.dart';
+import '/widgets/app_bar.dart';
+import '/repositories/local/genre_repository_class.dart';
+import '/widgets/back_button.dart';
+import '/widgets/description.dart';
+import '/widgets/header.dart';
+import '/model/movie.dart';
+import '/widgets/subheader.dart';
+import '/shared/constants.dart';
+
+class MovieDetail extends StatefulWidget {
+  const MovieDetail({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+
+  @override
+  State<MovieDetail> createState() => _MovieDetailState();
+}
+
+class _MovieDetailState extends State<MovieDetail> {
+  late int _likeCounter;
+  final GenreJsonManagement genreJsonManagement = GenreJsonManagement();
```

Comment:
Same as above

Reply:
For the moment I need to leave it like it is. If I change it, there will appear an error because when I call the method getNameFromIds, an object of the class IMyRepository couldn`t access to it. For the next PR I will change it